theme/debugger.less

```diff
@@ -203,6 +203,31 @@
     }
 }
 
+.debugger-preview {
+    position: fixed;
+    background-color: white;
+    color: black;
+    border: solid 1px black;
+    padding: 0.25rem;
+
+    -webkit-touch-callout: text; /* iOS Safari */
+    -webkit-user-select: text;   /* Chrome/Safari/Opera */
+    -khtml-user-select: text;    /* Konqueror */
+    -moz-user-select: text;      /* Firefox */
+    -ms-user-select: text;       /* Internet Explorer/Edge */
+    user-select: text;           /* Non-prefixed version, currently
```

Comment:
for what it's worth this seems to be supported across all browsers (with a note on ios): https://caniuse.com/#feat=mdn-css_properties_user-select_text no harm leaving vendor prefixes in though

Reply:
Copy/pasted from elsewhere in the codebase